test/export/fake-reader/export_config.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: ConfigMap
```

Comment:
Are there still plans to move this to a CRD to get the observed perf improvements?

Reply:
Yip! There are still plans to move this to a CRD.